examples/browser-angular/.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+**IMPORTANT: Please do not create a Pull Request for this repository.**
+
```

Comment:
It would be good to also say *why* the user should not open a PR for this repo, words similar to:

```suggestion

The contents of this repository are automatically synced from the parent [IPFS Examples Project](https://github.com/ipfs-examples/js-ipfs-examples) so any changes made here will be lost after the next sync.

Please open a PR against [IPFS Examples](https://github.com/ipfs-examples/js-ipfs-examples) instead.

```

This should be applied to all `pull_request_template.md` files in this PR.